### ISSUE TITLE
Token external links

### ISSUE THREAD

**Issue by rorygeoghegan** [reservoirprotocol/sample-marketplace#164]
For each token, we return the source of the best order, with this info:

```
"id": "0x5b3256965e7c3cf26e11fcaf296dfc8807c01073",
"name": "OpenSea",
"icon": "https://opensea.io/static/images/logos/opensea.svg",
"url": "https://opensea.io/assets/0x8d04a8c79ceb0889bdd12acdf3fa9d207ed3ff63/922"
```

It will be nice to show this and link to it, especially as we add more marketplaces.

Similarly, it might be nice to have an external link to etherscan on the token page

**Comment by ptrwtts** [comment]
@rorygeoghegan can we just quickly add an Etherscan and Open Sea link to the token page? Just chuck some icons somewhere, doesn't need design (can always change later). I use the sample marketplace for debugging, and it sucks to not have this.

Then separately, we will want to incorporate source information, which is more tied to the listing, rather than the token. But this is less urgent and still blocked by backend.

**Comment by rorygeoghegan** [comment]
Duplicate
